guide/taxonomy-ingestion.md

```diff
@@ -13,54 +13,45 @@ role in the integration is.
 
 | Term                       | Meaning |
 | -------------------------- | ------- |
-| **Attribute Spec**         | AttributeSpec is the specification of an Attribute within a Category on your channel. |
-| **Category**               | Categories provide a (usually) high-level classification for a Listing. Categories can be arranged in a tree-like structure with parent and child categories. The category will often be a crucial piece of information about how the data is structured when sending to the channel. For example, on Amazon it dictates which path to take down the XSD when constructing feed messages. Attribute Specs are associated directly with Categories. |
-| **Product Type**           | ProductType are detailed classifications dictated by the channel. They should be at least as specific as the Category. Generally they should be more specific than the Category. Product Types are used to specify which attributes are required and recommended for a given Listing. They are not relevant to the structure of the data that we are capable of sending. If this is not a concept in your system, you should create a product type for each category. |
-| **Quality Attribute Spec** | QualityAttributeSpec provides a connection between a ProductType and an AttributeSpec. If a Quality Attribute Spec exists for an AttributeSpec in a particular ProductType, that Spec is either Recommended or Required in that ProductType (and the Quality Attribute Spec will indicate which it is). |
+| **Attribute**    | An Attribute is the specification (name, type, product type etc...) of an Attribute within a ProductType on your marketplace. |
+| **Product Type** | ProductType are detailed classifications dictated by the marketplace. Product Types are used to specify which attributes are available, required and recommended for Listing that are classified into them. |
 
-## Creating Categories
-
-You should construct the Category tree first.
-All of the other objects will depend in one way or another on a Category.
+## Product Types
 
-Categories can be created using the `CreateCategory` action:
+The first step in building a Taxonomy is to populate your ProductTypes.
+To create ProductTypes, use the `CreateProductType` action:
 
 ```
-POST /v2/storefront/taxonomy/category
+POST /v2/storefront/taxonomy/product_type
 ```
 
-With an example request like:
+with a request like:
 
 ```json
 {
-  "category": {
-    "channelCategoryId": "<an identifier for your channel, must be unique>",
-    "parentCategoryId": "<the ID of the parent category, can be omitted or set to an empty string to indicate root level>",
-    "displayName": "<a user friendly display name>"
+  "productType": {
+    "productTypeId": "<the productType ID to associate the product type with>",
+    "channelProductTypeId": "<your id for this product type>"
```

Comment:
what's the difference between this and productTypeId?

Reply:
that shouldnt be there, just fixed it to be:

```

```json
{
  "productType": {
    "productTypeId": "<a unique identifier for this product type>",
    "displayName": "< a user-friendly name for this product type>"
  }
}
```